tests/conftest.py

```diff
@@ -33,16 +36,18 @@ def _dataset(
         width: int = 256,
         height: int = 256,
     ):
+        max_value = 127 if dtype == "int8" else 255
+
         # Data
         arr = numpy.zeros((nband, height, width), dtype=dtype) + 1
+        arr[:, range(height), range(width)] = max_value
+        arr[:, range(height - 1, 0, -1), range(width - 1)] = max_value
+        arr[:, :, width // 2] = max_value
+        arr[:, height // 2, :] = max_value
```

Comment:
we add some values 
<img width="1000" alt="Screenshot 2024-10-04 at 12 44 15 PM" src="https://github.com/user-attachments/assets/4cd51589-9ac4-4f53-9407-2e48b15af293">